onebusaway-android/src/main/java/org/onebusaway/android/report/ui/dialog/CustomerServiceDialog.java

```diff
@@ -123,11 +123,14 @@ protected void initView(View view, ObaAgencyWithCoverage coverage) {
                     public void onClick(View view) {
                         String locationString = getActivity().getIntent().
                                 getStringExtra(BaseReportActivity.LOCATION_STRING);
-                        String agencyName = "\nAgency: " + agency.getName();
-                        UIUtils.sendEmail(getActivity(), agency.getEmail(), locationString + agencyName);
+                        UIUtils.sendEmail(getActivity(), agency.getEmail(), locationString);
+
+                        String gaEventLabel = locationString != null ?
```

Comment:
Fixed in this commit https://github.com/OneBusAway/onebusaway-android/pull/392/commits/2e130cac93ed9a1aad285c04e51f77fb3f4777e3.